src/bus/policy.c

```diff
@@ -835,6 +846,14 @@ int policy_snapshot_check_own(PolicySnapshot *snapshot, const char *name_str) {
         size_t i;
         int v, r;
 
+        r = bus_apparmor_check_own(snapshot->apparmor, snapshot->seclabel, name_str);
+        if (r) {
+                if (r == BUS_APPARMOR_E_DENIED)
+                        return POLICY_E_APPARMOR_ACCESS_DENIED;
+
```

Comment:
Are there other return values that mean "don't grant access"? eg. "internal error" or "silently deny" or similar? This currently feels like a 'fail open' way to write this check. (And two more later.)

Reply:
This is a translation of error codes from one domain to another. If we happen to forget converting one, the result is the following `error_fold()`, which takes any unhandled error and turns it into a fatal error code. Hence, the worst that can happen is that we end up exiting, rather than just refusing the message-transaction. Not good, but also we do not accidentally fail the security check.